### ISSUE TITLE
Can cycle lanes be rendered when they are in the middle of other lanes (using  cycleway:lanes)?

### ISSUE THREAD

**Issue by momu** [cyclosm/cyclosm-cartocss-style#590]
In my area a street was rebuilt, I updated it in `OSM` and now I ask myself if there is already a solution for cycle lanes which are in the middle of other lanes. 

It looks like this: 

![https://wiki.openstreetmap.org/w/images/thumb/c/cc/Kreuzung_mit_Radspur.jpg/800px-Kreuzung_mit_Radspur.jpg](https://wiki.openstreetmap.org/w/images/thumb/c/cc/Kreuzung_mit_Radspur.jpg/800px-Kreuzung_mit_Radspur.jpg)

See for details:

https://wiki.openstreetmap.org/wiki/Lanes#Crossing_with_a_designated_lane_for_bicycles

There is also a link to additional examples.

In the wiki it is proposed to tag the picture above like this:
```
lanes=3
 turn:lanes=left|through|through|right
 vehicle:lanes=yes|yes|no|yes
 bicycle:lanes=yes|no|designated|yes
 cycleway:lanes=no|no|lane|no
```
It all boils down to  using `cycleway:lanes`. So are there are any plans to support these tags to render the lanes in the middle of others?

**Comment by Florimondable** [comment]
There are 6800 usage for bicycle:lane + 7000 forward/backward variant, we could think of rendering them.

https://taginfo.openstreetmap.org/search?q=bicycle%3Alanes

**Comment by momu** [comment]
> we could think of rendering them.

That would be nice.

**Comment by VladimirMorozov** [comment]
Shouldn't it be also tagged with cycleway=lane which would render that a bicycle lane exists?
This is a case for a one-way road, but would be incorrect for a two-way road with such a lane though. So rendering it would be useful. But when checking the usage statistics, I think that only those which don't have cycleway/cycleway:left/cycleway:right should be counted.

Also, if a new way to display it on a map has to be created, I just wanted to add that it may get complicated.
Sometimes road planners get overly creative, here are two bicycle lanes and maybe both of them could be counted as being in the middle of the road (the one on the bottom actually has a lane for a bus stop to the right). 
![a](https://user-images.githubusercontent.com/2279813/125488726-fb5fd767-e1fd-45dd-aa26-7288b8088ad8.png)
(aerial imagery by Maaamet)

**Comment by momu** [comment]
> Shouldn't it be also tagged with cycleway=lane which would render that a bicycle lane exists?

No, I don't think so, as ` cycleway:lanes` or `bicycle:lanes` covers it.

In the picture above  `bicycle:lanes=no|designated|no|no|no|designated` should do it.

**Comment by Florimondable** [comment]
> Also, if a new way to display it on a map has to be created, I just wanted to add that it may get complicated.

Yes, it'd be complicated, it's not in my priority list ;)

**Comment by matkoniecz** [comment]
> Shouldn't it be also tagged with cycleway=lane which would render that a bicycle lane exists?

Yes, or one of its variants.

> No, I don't think so, as cycleway:lanes or bicycle:lanes covers it.

Tagging with both is OK, similarly like combining `busway=lane` with `psv:lanes=yes|yes|designated` or `bus:lanes=yes|yes|designated` is 100% fine.
